.travis.yml

```diff
@@ -0,0 +1,18 @@
+sudo: false
+language: python
+install: pip install tox
+matrix:
+  include:
+    - python: "2.6"
+      env: TOX_ENV=py26
+    - python: "2.7"
+      env: TOX_ENV=py27
+    - python: "3.6"
+      env: TOX_ENV=py36
+    - python: "2.7"
+      env: TOX_ENV=static
```

Comment:
I think 'static' including black is only dealing with formatting and code style checks and doesn't affect the actual execution. So it should be fine to run this python3, unless there is some idea I'm missing on.

Reply:
Static checks would pick up python3 incompatibilities, though, right? That's not particularly helpful before the module supports python3.

Reply:
As for black specifically, it will fail to parse a file that's incompatible with python3.

Reply:
Alright. Ignore this comment then.

Reply:
We can, @rajulkumar . It can be added back with future python3 compatibility changes to the script.

Also, I'm quite sure static checks will block all PRs until alt-src is heavily refactored. *Maybe* it would work on python2.4 but that can't be achieved.

Reply:
So, basically, we need to whittle tests down to just py26, py27, and cov/cov-travis.